packages/commercetools/api-client/src/helpers/search/index.ts

```diff
@@ -77,7 +77,7 @@ const buildProductWhere = (settings: Config, search: ProductWhereSearch) => {
     predicates.push(`id in ("${search.ids.join('","')}")`);
   }
```

Comment:
```suggestion
  if (search?.sku) {
    predicates.push(`sku="${search.sku}"`);
  }
```

From what I've understood from #6181 I think we need to add this rule

Reply:
But please test if this makes sense and if is the best way.

Reply:
@alefbarbeli `search.skus` are passed directly to gql query as variable, 
https://github.com/vuestorefront/vue-storefront/blob/1d0e68c437ecbee664d431b93f730c0bde228485/packages/commercetools/api-client/src/api/getProduct/index.ts#L17

This Error is caused by gql `where` variable passed to query as empty string

Reply:
Understood! Well done